examples/forecasting_strategies.ipynb

```diff
@@ -0,0 +1,954 @@
+      "text/plain": [
+       "<Figure size 720x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "from etna.datasets import TSDataset\n",
+    "\n",
+    "original_df = pd.read_csv(\"data/monthly-australian-wine-sales.csv\")\n",
+    "original_df[\"timestamp\"] = pd.to_datetime(original_df[\"month\"])\n",
+    "original_df[\"target\"] = original_df[\"sales\"]\n",
+    "original_df.drop(columns=[\"month\", \"sales\"], inplace=True)\n",
+    "original_df[\"segment\"] = \"main\"\n",
+    "original_df.head()\n",
+    "df = TSDataset.to_dataset(original_df)\n",
+    "ts = TSDataset(df=df, freq=\"MS\")\n",
+    "ts.plot()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HORIZON = 14"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Strategies\n",
+    "\n",
+    "### 2.0 What are strategies\n",
+    "\n",
+    "In most cases to make forecasts you should:\n",
+    "\n",
+    "1. Do some transformations with data\n",
+    "2. Fit model/some models\n",
+    "3. Get predictions from your model\n",
+    "\n",
+    "The ETNA library has several tools to optimize these 3 steps. Let's look at each of them:\n",
```

Comment:
This notebook is about forecasting strategies, not about all the tools of out library)))
I guess this intro is slightly irrelevant here, may be it is better to say that there are several possible forecasting strategies available in ETNA and this section will examine them closely.